src/js/smooch.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { render, unmountComponentAtNode } from 'react-dom';
 import pick from 'lodash.pick';
+import { batchActions } from 'redux-batched-actions';
 
 import { store } from './stores/app-store';
```

Comment:
Yeah the idea is that with code splitting, loading the store could be done asynchronously only if a Smooch method needs it or when it needs to render. Entry point should be as light as possible. When I tried this last, I remember that since the store was required in a lot of places, the bundles webpack was generating were not really worth it.